### DISCUSSION TITLE
Welcome to Bigcommerce Discussions!

### DISCUSSION THREAD

**Discussion by Yooler2675** [Yooler2675/Bigcommerce#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kwoods1981** [comment]
Hello all!

I'd love to connect with other webdevs using BigCommerce for their online store. I own [Bitty Bows](www.bittybowsboutique.com) and have been with BC for about 3 years, I'm using the Chiara theme. My husband works in WebDev and has done all the customizations in my store. I'd love to see what other stores have done to get ideas! I'd be happy to add you to our repository to see what we've done too. Thanks in advance!
